src/ocaml/cm_kind.mli

```diff
@@ -9,6 +9,8 @@ val compare : t -> t -> Ordering.t
 
 val all : t list
 
+val choose : 'a -> 'a -> 'a -> t -> 'a
```

Comment:
Is this function used anywhere?

Reply:
nope, I used it at some point but then wasn't needed. removed in https://github.com/jchavarri/dune/pull/4/commits/7854dddcaeaf468a38dcff9be5d4531061ada6fc.